### DISCUSSION TITLE
Re-Activate Backtesting.Py

### DISCUSSION THREAD

**Discussion by IndiasFernandes** [kernc/backtesting.py#800] [Ideas]
Hello there, I tried to find a way to contact you (kernc) directly,but didn't find any way so decided to leave this here.

I've been using backtesting.py quite regularly latelly and would love to connect to this community to help to develop it further. Isthere a discord channel or any other way to be in contact and to ask some questions/learn before developing and pushing anything?

Thanks and congrats for the project

**Comment by kernc** [comment]
Well, we have this discussion forum and the issue tracker for general framework related questions and discussion. :smiley:
Other than that, an email might be available, signing some commits.
